cluster/config-defaults.yaml

```diff
@@ -703,3 +703,6 @@ deployment_service_enabled: "true"
 
 # Standard storageclass gp3 volume type
 storageclass_standard_gp3: "false"
+
+# default for ZMON is not to be able to get any objects from S3
+zmon_s3_allow_get_object: ""
```

Comment:
How about we make the default "false" similar to the `storageclass_standard_gp3` setting above.

Reply:
Nevermind, I see now that it's a list of buckets and not a bool.

Reply:
Then let's rename the field to something like: `zmon_accessible_s3_buckets` and add add a comment that states that it's a list of buckets separated by comma.